static/app/views/performance/transactionSummary/transactionOverview/content.tsx

```diff
@@ -344,6 +383,75 @@ function SummaryContent({
     organization.features.includes('performance-spans-new-ui') &&
     organization.features.includes('insights-initial-modules');
 
+  // This is adapted from the `getEventFieldValues` function in `events/searchBar.tsx`
+  const getTransactionFilterTagValues = useCallback(
+    async (tag: Tag, queryString: string) => {
+      const projectIdStrings = eventView.project?.map(String);
+
+      if (isAggregateField(tag.key) || isMeasurement(tag.key)) {
+        // We can't really auto suggest values for aggregate fields
+        // or measurements, so we simply don't
+        return Promise.resolve([]);
+      }
+
+      // device.class is stored as "numbers" in snuba, but we want to suggest high, medium,
+      // and low search filter values because discover maps device.class to these values.
+      if (isDeviceClass(tag.key)) {
+        return Promise.resolve(DEVICE_CLASS_TAG_VALUES);
+      }
+
+      try {
+        const results = await fetchTagValues({
+          api,
+          orgSlug: organization.slug,
+          tagKey: tag.key,
+          search: queryString,
+          projectIds: projectIdStrings,
+          includeTransactions: true,
+          includeSessions: true,
+          sort: '-count',
+          endpointParams: {start: eventView.start, end: eventView.end},
```

Comment:
This is missing `statsPeriod`, so if you have say 14d selected this doesn't limit by time range